### DISCUSSION TITLE
"Summary event of the report's signatures" alert dashboard gives and error "rison decoder error: missing ':'"

### DISCUSSION THREAD

**Discussion by facyber** [Security-Onion-Solutions/securityonion#4308] [Unsupported Versions] [answered]
Hello everyone,

Today I noticed quite a number of "Summary event of the report's signatures"  alerts and I wanted to investigate a bit in Kibana, but when I opened alerts view and then clicked on "Summary event of the report's signatures", it opened a new but but it reproduced the error bellow:

```
Error restoring state from URL

rison decoder error: missing ':'

errcb@https://somanagerhost/kibana/38015/bundles/plugin/kibanaUtils/kibanaUtils.plugin.js:1:51969
rison.parser.prototype.parse@https://somanagerhost/kibana/38015/bundles/plugin/kibanaUtils/kibanaUtils.plugin.js:1:52713
rison.decode@https://somanagerhost/kibana/38015/bundles/plugin/kibanaUtils/kibanaUtils.plugin.js:1:52111
decodeState@https://somanagerhost/kibana/38015/bundles/plugin/kibanaUtils/kibanaUtils.plugin.js:1:132809
getStatesFromKbnUrl/<@https://somanagerhost/kibana/38015/bundles/plugin/kibanaUtils/kibanaUtils.plugin.js:1:137702
getStatesFromKbnUrl@https://somanagerhost/kibana/38015/bundles/plugin/kibanaUtils/kibanaUtils.plugin.js:1:137669
getStateFromKbnUrl@https://somanagerhost/kibana/38015/bundles/plugin/kibanaUtils/kibanaUtils.plugin.js:1:137803
get@https://somanagerhost/kibana/38015/bundles/plugin/kibanaUtils/kibanaUtils.plugin.js:1:144327
syncQueryStateWithUrl@https://somanagerhost/kibana/38015/bundles/plugin/data/data.plugin.js:1:145494
useDashboardStateManager/<@https://somanagerhost/kibana/38015/bundles/plugin/dashboard/dashboard.chunk.1.js:1:183365
fs@https://somanagerhost/kibana/38015/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js:375:77781
El@https://somanagerhost/kibana/38015/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js:375:102507
__kbnSharedDeps__</t.unstable_runWithPriority@https://somanagerhost/kibana/38015/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js:383:3462
Hr@https://somanagerhost/kibana/38015/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js:375:45442
Sl@https://somanagerhost/kibana/38015/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js:375:102270
yl/<@https://somanagerhost/kibana/38015/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js:375:102181
U@https://somanagerhost/kibana/38015/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js:383:2845
__kbnSharedDeps__</T.port1.onmessage@https://somanagerhost/kibana/38015/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js:383:1589
```

I can do a workaround by opening different rule and then editing the filter with the name of this alert and that would work, but it does not work by clicking on it from Alert dashboard. I tested with multiple others and didn't noticed this issue, only with this one, so I am not sure if this is a bug or maybe some data is corrupted.

I checked the logs for Kibana and Nginx but I don't see any error messages there.

Setup: Manager + 2x Heavy nodes
SO version: 2.3.51

**Comment by dougburks** [comment, marked as answer]
The `Summary event of the report's signatures` alert appears to be generated by the following Wazuh rule in the Windows ruleset `/opt/so/rules/hids/ruleset/rules/0585-win-application_rules.xml`:
```
  <rule id="60608" level="4">
    <if_sid>60600</if_sid>
    <field name="win.system.providerName">^Windows Error Reporting$</field>
    <field name="win.system.eventID">^1001$</field>
    <description>Summary event of the report's signatures</description>
    <options>no_full_log</options>
  </rule>
```

You could try accessing this alert from our other non-Kibana interfaces like [Alerts](https://docs.securityonion.net/en/2.3/alerts) and [Hunt](https://docs.securityonion.net/en/2.3/hunt) to see what happens there.

You might also try overwriting rule `60608` with a new rule that changes `report's` to `reports` to see if the apostrophe might be causing problems:
https://docs.securityonion.net/en/2.3/wazuh.html#tuning-rules

Finally, if you look at the alert and realize that you don't need it, you could just disable it by changing `level` from `4` to `0`.

**Reply by facyber** [reply]
Do I need to use after adding rule `sudo salt \* state.highstate` ?

Thanks for the info, I will try that now and we will see. We disabled Application logs from Wazuh for that specific host who was generating those alarms, but will revert it back.
